python/cuml/cuml/internals/base.pyx

```diff
@@ -655,7 +650,9 @@ class UniversalBase(Base):
         """Transfer attributes from GPU estimator to CPU estimator."""
         for name in self.get_attr_names():
             try:
-                value = getattr(self, name)
+                # This avoids calling `__getattr__`, which may recurse
+                # in cuml.accel models.
+                value = object.__getattribute__(self, name)
```

Comment:
Why was that previously not an issue?

Reply:
It was. Before this was handled by temporarily disabling `cuml.accel` when resolving an attribute (see [here](https://github.com/rapidsai/cuml/blob/dc6a647905022fed772bfd5a0452b3231089b2d6/python/cuml/cuml/internals/base.pyx#L846-L852)).

The recursion happens in `gpu_to_cpu()`, and only for hdbscan (at least in our tests). I suspect this is something wonky in how that estimator is written, but in general just skipping `__getattr__` here should be fine and avoids the recursion issue. Once we're using composition for the proxies this should go away completely anyway.